airflow-ctl/docs/howto/index.rst

```diff
@@ -66,15 +77,36 @@ In both cases token is securely stored in the keyring backend. Only configuratio
 is the API URL and the environment name. The token is stored in the keyring backend and is not persisted in the
 configuration file. The keyring backend is used to securely store the token and is not accessible to the user.
 
-Datetime Usage
-''''''''''''''
-For datetime parameters, date should be timezone aware and in ISO format.
-For example: ``2025-10-10T10:00:00+00:00``
-Let's take example of triggering a DAG run with a logical date, run after and a note.
+What is authentication for ``airflowctl``?
+``````````````````````````````````````````
+For ``airflowctl`` to be able to communicate with the Airflow API, it needs to authenticate itself and acquire token.
+This is done using either a token or a username and password.
+The token can be acquired from the Airflow API or generated using a username and password.
 
-.. code-block:: bash
+.. image:: ../images/diagrams/airflowctl_api_network_diagram.svg
```

Comment:
Nit: how thie image was prepared ? What if we need to modify them? 

I am quite a fan of creating such images using python code (like airflow DAGs) - with `diagram` package. We have a few of those architecture diagrams generated in our docs and we even have prek hook that will automatically re=generate the images when needed:

For example: https://github.com/apache/airflow/blob/main/docs/images/documentation_architecture.py

* https://github.com/apache/airflow/blob/main/scripts/ci/prek/generate_airflow_diagrams.py

This way anyone in the future can modify the sources for such diagrams and re-run the generator to get the image reganerated...

Maybe we could use it here too ?